app/controllers/appointments_controller.rb

```diff
@@ -38,10 +38,12 @@ def index
 
   def update
     call_result = appointment_params[:call_result].to_sym
+    search_filters = appointment_params[:search_filters]&.split || []
```

Comment:
As best I can tell, the only way to include an array in a hidden field is as a space separated string, so we have to split here.